.github/workflows/ci-publish-base-image.yml

```diff
@@ -2,7 +2,7 @@ name: Publish Base CI Image To Dockerhub
 
 on:
   push:
-    branches: [main, master, release-*, gha-test-*]
```

Comment:
Is this PR not handling the building of docker images on release?

Reply:
no, this produces the base image for CI.   It was overbuilding (gha-test-*) is only needed when testing a change here, and release-* doesn't matter because we are not maintaining separate base images for CI in different repos.

Reply:
Huh need to escape the stars I guess....  \* maybe?

Reply:
hah \* worked \*